modules/sdk-core/src/bitgo/lightning/lightning.ts

```diff
@@ -24,7 +24,14 @@ export class Lightning implements ILightning {
   }
 
   public async createInvoice(params?: CreateInvoiceParams): Promise<CreateInvoiceResponse> {
```

Comment:
Want to make sure. `value` is set as required but the params are optional. Is this intended?

Reply:
The params should be required. Fixed. Thank you!